chapters/introduction.tex

```diff
@@ -76,7 +76,9 @@ \section{Some Definitions}\label{some-definitions}
 % - translation (for phrases such as "during translation")
 % - deprecated
 % - quality of implementation
-% - simulator
+%
+% Note used:
+% - simulator (the program that computes the simulation)
```

Comment:
I guess you meant _not_, but I'd even prefer to not include commented out definitions at all.  For an example of why, at Wolfram we tend to call the program that computes the simulation result the _simulation_, so what is said here in the comment doesn't match the terminology we use.  Hence, I think that all definitions we use should be included in the actual document, forcing us to reach agreement on at least those terms.

```suggestion
```